code/modules/mob/inventory.dm

```diff
@@ -188,19 +188,41 @@
 	drop_r_hand()
 	drop_l_hand()
 
-//drop the inventory item on a specific location
-/mob/proc/transferItemToLoc(obj/item/I, atom/newloc, nomoveupdate, force)
-	return UnEquip(I, newloc, nomoveupdate, force)
+/**
+  * Used to drop an item (if it exists) to the ground.
+  * * Will return TRUE is successfully dropped.
+  * * Will return FALSE if the item can not be dropped due to TRAIT_NODROP via doUnEquip().
+  * * Will return null if there is no item.
+  * If the item can be dropped, it will be forceMove()'d to the ground and the turf's Entered() will be called.
+*/
+/mob/proc/dropItemToGround(obj/item/I, force = FALSE)
+	. = UnEquip(I, force, drop_location())
+	if(.)
+		I.pixel_x = rand(-6,6)
+		I.pixel_y = rand(-6,6)
```

Comment:
Is this ever reset? Could this lead to +6 every drop until eventually the item is off the turf?

Would `I.pixel_y = initial(I.pixel_y) + rand(-6,6)` or something be better?

Reply:
Good catch, added a pixel-offset resetting when put on hands. It works properly now.